libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/GetConversationMembers.cs

```diff
@@ -74,13 +75,7 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
                 return await dc.EndDialogAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
             }
 
-            var bfAdapter = dc.Context.Adapter as BotFrameworkAdapter;
-            if (bfAdapter == null)
-            {
-                throw new InvalidOperationException("GetConversationMembersAsync() only works with BotFrameworkAdapter");
-            }
-
-            var result = await bfAdapter.GetConversationMembersAsync(dc.Context, cancellationToken).ConfigureAwait(false);
+            var result = await TeamsInfo.GetMembersAsync(dc.Context, cancellationToken).ConfigureAwait(false);
```

Comment:
I think GetConversationMembersAsync is NOT Teams specific.

TeamsInfo.GetMembersAsync will return TeamsChannelAccounts. Maybe this is okay?

![image](https://user-images.githubusercontent.com/11055362/131884407-ba48865c-2a26-42f4-a9c0-f67dc9f072b3.png)

Reply:
I'll rework this to return as it did in the old code in that it returned just ChannelAccount and not TeamsChannelAccount objects.